ando/tools/generator/utils.py

```diff
@@ -0,0 +1,84 @@
+from os import path
+import pandas as pd
+import os
+import json
+
+
+def save_tsv(dataframe, path_to_save):
+    """
+
+    Parameters
+    ----------
+    dataframe: dataframe
+        dataframe to save in a TSV format
+    path_to_save: str
+        path to save the TSV file
+
+    Returns
+    -------
+
+    """
+    if path.exists(path_to_save):
+
+        df = pd.read_csv(os.path.join(path_to_save), sep='\t')
+        output = df.append(dataframe, sort=True)
+        output.to_csv(path_to_save, sep="\t", index=False)
+
+    else:
+        dataframe.to_csv(path_to_save, sep="\t", index=False)
+
+
+def save_json(new_dict, path_to_save):
+    """
+
+    Parameters
+    ----------
+    new_dict: dict
```

Comment:
The term `new_dict` does not make sense here for the user as it is the only dict he knows about. Maybe `data_dict` would be better?

Reply:
done